### DISCUSSION TITLE
Codespaces and vscode.dev not loading

### DISCUSSION THREAD

**Discussion by 200652** [community/community#63015] [Codespaces]
### Select Topic Area

Bug

### Body

Hello, I just wanted to start by prefacing that I am very new to coding and had started with Harvards CS50 course and am currently on lecture 1. 

I had been using GitHub’s Codespaces and vscode(web) to work through the problem sets for a while, but recently I had  travelled for nearly a month and had left my device(my iPad) at my home and had yet to reopen everything on Codespaces and vscode.

After I came back every time I tried opening my Codespaces on the web it always gives me a blank screen with either “setting up your web editor” or “setting up your Codespace” and never loading from there and staying static.

This might be an issue regarding my device(an iPad) as I have been able to access my files on vscode on my phone which work just fine, however I had done nothing with my iPad for nearly a month so I am unsure as to why nothing is loading at all.

![DF6BBF60-4B37-498B-8328-28027FC0C0B9](https://github.com/community/community/assets/137102237/b9b266fe-30cd-4978-870d-243a3616d9ce)

**Comment by 200652** [comment]
![E1DB843C-0DAB-4988-80B9-063627BF1157](https://github.com/community/community/assets/137102237/54519fa1-31d7-44f8-adc0-56f1b1051100)
 
 vscode.dev doesn’t work either.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @200652! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by coltonhugg** [comment]
Having the same problem here, using a VPN does not help for both Codespaces and viscose.dev

**Reply by 200652** [reply]
Ended up fixing it soon enough.
Seemed to have been an issue regarding storage and data on my ipad, and also software updates that hadn’t been done, although Occasionaly vscode.dev still bugs out so the issue might not be fully tuned in.

**Reply by plemnoooooo** [reply]
+1. Works after updating it. Seems to be the only workaround, probably because the latest updates of the codespaces is using some new feature that slightly older versions don't have

**Reply by robertandrews** [reply]
Please can you clarify the fix steps?

I’d rather not update my iPadOS 15.6.1 - I’d rather the team make the web app compatible. It used to work fine. Now I can’t access either github.dev or vscode.dev on iPad.

**Comment by plemnoooooo** [comment]
Same here. Doesn't work on iPad while it works fine anywhere else.

**Reply by J4KU** [reply]
Upgrade the iPadOS 😉

**Comment by conradwt** [comment]
I'm seeing a similar issue.  For example,

1. Create GitHub Codespace
2. I see "Launching Visual Studio Code" message

![Screenshot 2023-07-30 at 11 21 01 PM](https://github.com/microsoft/vscode/assets/20219/f9e190ca-4a34-45df-838b-640399ccefbe)

**Reply by plemnoooooo** [reply]
Check that you have installed VS Code for **Desktop**. It should open it just fine with the repository you are working on.

If you don't have it, you will need to install it or configure it to open VS Code for the **Web**.

Otherwise, create a new issue, since it is a different bug/glitch altogether that happens on different devices.

**Comment by robertandrews** [comment]
Also broken for me on iPadOS :-(

https://github.com/microsoft/vscode/issues/190930

https://github.com/orgs/community/discussions/67161

**Reply by plemnoooooo** [reply]
You will have to update your iPad because there are some web features used in Codespaces that are unavailable in older versions of iPadOS.

**Reply by robertandrews** [reply]
I’d rather not. It used to work fine (iPadOS 15.6.1).

**Reply by plemnoooooo** [reply]
Then I am afraid to say that you aren't going to get the codespaces to run properly anytime soon. The only solution is to update your iPad, since the features that are used in codespaces need new functionality that is only available in later updates.

**Comment by iKunalmathur** [comment]
did you guys find any solution ?? i'm having the same issue in firefox

**Reply by robertandrews** [reply]
No :-((( And now ChatGPT.com no longer works on my iPadOS 15.6.1 Safari either.
